src/AlgAssAbsOrd/Order.jl

```diff
@@ -849,11 +849,11 @@ function MaximalOrder(O::AlgAssAbsOrd{S, T}) where { S <: AlgGrp, T <: AlgGrpEle
       if mod(d, p^2) != 0
```

Comment:
can overflow

Reply:
This `p` is a factor of `degree(O) = dim(algebra(O))`. Can this really become larger than $2^{31}$? The `p` in the `p^2` above that is a factor of `discriminant(O::AlgAssAbsOrd) :: ZZRingElem`.

Reply:
If it really can't overflow, it'd still be useful to add a comment stating (and justifying) that

Reply:
Added an assertion